python/tests/test_pyhdk_api.py

```diff
@@ -130,6 +130,36 @@ def test_import_csv(self, header, create_table, full_schema, glob):
 
         hdk.drop_table(table_name)
 
+    @pytest.mark.parametrize("create_table", [True, False])
+    @pytest.mark.parametrize("glob", [True, False])
+    def test_import_parquet(self, create_table, glob):
+        hdk = pyhdk.init()
+        table_name = "table1"
+
+        if create_table:
+            hdk.create_table(table_name, (("a", "int64"), ("b", "fp64")))
+            real_schema = {"a": "INT64", "b": "FP64"}
+
+        if glob:
+            file_name = ["omniscidb/Tests/ArrowStorageDataFiles/int_float.parquet", "omniscidb/Tests/ArrowStorageDataFiles/int_float.parquet"]
+            ref_data = {
+                "a": [*range(1, 6), *range(1, 6)],
+                "b": [1.1, 2.2, 3.3, 4.4, 5.5, 1.1, 2.2, 3.3, 4.4, 5.5],
+            }
+        else:
+            file_name = "omniscidb/Tests/ArrowStorageDataFiles/int_float.parquet"
+            ref_data = {"a": [*range(1, 6)], "b": [1.1, 2.2, 3.3, 4.4, 5.5]}
+
+        if not create_table:
```

Comment:
Either move it up to the previous `create_table` if-statement or define schema unconditionally